.licenserc.yaml

```diff
@@ -23,28 +24,20 @@ header:
     spdx-id: N/A
     copyright-owner: DataRobot, Inc.
     content: |
-
```

Comment:
Curious why the whitespace changed here -was it just incorrect before

Reply:
You can define your own rules in `pyproject.toml`
https://lyz-code.github.io/yamlfix/#comments-whitelines
I have only set up basic rules there.